tests/testsuite/profile_trim_paths.rs

```diff
@@ -530,14 +567,14 @@ fn object_works_helper(run: impl Fn(&std::path::Path) -> Vec<u8>) {
         .with_stderr(&format!(
             "\
 [COMPILING] bar v0.0.1
-[RUNNING] `rustc [..]\
+[RUNNING] `rustc [..]-C split-debuginfo={split_debuginfo} [..]\
     -Zremap-path-scope=object \
     --remap-path-prefix={pkg_remap} \
     --remap-path-prefix=[..]/lib/rustlib/src/rust=/rustc/[..]
 [COMPILING] foo v0.0.1 ([CWD])
-[RUNNING] `rustc [..]\
+[RUNNING] `rustc [..]-C split-debuginfo={split_debuginfo} [..]\
     -Zremap-path-scope=object \
-    --remap-path-prefix=[CWD]= \
+    --remap-path-prefix=[CWD]=. \
```

Comment:
This looks like a result I'd expect from the follow up commit...

Reply:
fixed 😆